sisyphus/global_settings.py

```diff
@@ -392,5 +392,6 @@ def update_global_settings_from_env():
 # Set to False to disable Warning of unset engine
 ENGINE_NOT_SETUP_WARNING = True
 
-update_global_settings_from_env()
+update_global_settings_from_file(GLOBAL_SETTINGS_FILE_USER)
 update_global_settings_from_file(GLOBAL_SETTINGS_FILE_DEFAULT)
+update_global_settings_from_env()
```

Comment:
This order change might have an effect on some setups that make use of setting global settings from env, but I think most people are not doing it.

Reply:
I think it was an accident that env was before file. The main use cases of env is to overwrite settings without changing the file, e.g. testing something in a directory that doesn't belong to you.